src/types.jl

```diff
@@ -67,21 +74,31 @@ immutable Planet
     ecc::Float64
     axis::Float64
     period::Float64
-    Planet(name, radius, eqradius, polradius, mass, ecc, axis, period) =
+    inc::Float64
+    asc_long::Float64
+    per_long::Float64
+    mean_long::Float64
+    Planet(name, radius, eqradius, polradius, mass, ecc, axis, period, inc,
+           asc_long, per_long, mean_long) =
         new(String(name), Float64(radius), Float64(eqradius), Float64(polradius),
-            Float64(mass), Float64(ecc), Float64(axis), Float64(period))
+            Float64(mass), Float64(ecc), Float64(axis), Float64(period), Float64(inc),
+            Float64(asc_long), Float64(per_long), Float64(mean_long))
 end
 
 # New type representation
 function show(io::IO, pl::Planet)
-    println(io, "Planet:            ", ucfirst(pl.name))
-    println(io, "mean radius:       ", pl.radius, " m")
-    println(io, "equatorial radius: ", pl.eqradius, " m")
-    println(io, "polar radius:      ", pl.polradius, " m")
-    println(io, "mass:              ", pl.mass, " kg")
-    println(io, "eccentricity:      ", pl.ecc)
-    println(io, "semi-major axis:   ", pl.axis, " m")
-    print(io,   "period:            ", pl.axis, " s")
+    println(io, "Planet:                    ", ucfirst(pl.name))
+    println(io, "mean radius:               ", pl.radius, " m")
+    println(io, "equatorial radius:         ", pl.eqradius, " m")
+    println(io, "polar radius:              ", pl.polradius, " m")
+    println(io, "mass:                      ", pl.mass, " kg")
+    println(io, "eccentricity:              ", pl.ecc)
+    println(io, "semi-major axis:           ", pl.axis, " m")
+    println(io, "period:                    ", pl.period, " s")
+    println(io, "inclination                ", pl.inc, " θ")
```

Comment:
It's supposed to represent degrees. I forgot it's just a local interpretation used in my place. 
Just checked and found  `°`, I'll replace it with this.